pkg/ccl/logictestccl/testdata/logic_test/multi_region

```diff
@@ -534,8 +534,14 @@ GRANT admin TO testuser
 
 user testuser
 
-# Test adding a primary region to the system database.
-statement error adding a primary region to the system database is not supported
+# Test adding a primary region to the system database as a secondary tenant.
+onlyif config multiregion-9node-3region-3azs-tenant
+statement error pgcode 42501 user testuser may not modify the system database
+ALTER DATABASE system PRIMARY REGION "ap-southeast-2"
+
+# Note that nobody can add a region to the system database of the system tenant.
+skipif config multiregion-9node-3region-3azs-tenant
+statement error pgcode 0A000 modifying the regions of system database is not supported
```

Comment:
in CI this assertion failed with:
```
        expected:
        modifying the regions of system database is not supported
        got:
        pq: user testuser may not modify the system database
```

https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_BazelEssentialCi/9369070?showRootCauses=false&expandBuildChangesSection=true&expandBuildProblemsSection=true&expandBuildTestsSection=true

Reply:
filed https://github.com/cockroachdb/cockroach/issues/100404